Content.Server/Power/Pow3r/BatteryRampPegSolver.cs

```diff
@@ -54,7 +54,7 @@ public void Tick(float frameTime, PowerState state, IParallelManager parallel)
                 // suppliers + discharger) Then decide based on total layer size whether its worth parallelizing that
                 // layer?
                 _networkJob.Networks = group;
-                parallel.ProcessNow(_networkJob, group.Count);
+                parallel.ProcessSerialNow(_networkJob, group.Count);
```

Comment:
Really?

Reply:
Yeah, this is genuinely how you fix power flickers. Crazy, right?